subsys/bluetooth/host/hci_core.c

```diff
@@ -157,6 +143,12 @@ static struct cmd_data cmd_data[CONFIG_BT_HCI_CMD_COUNT];
 #define cmd(buf) (&cmd_data[net_buf_id(buf)])
 #define acl(buf) ((struct acl_data *)net_buf_user_data(buf))
 
+void bt_hci_cmd_data_state_set(struct net_buf *buf,
```

Comment:
maybe static inline this one in the header as well?

Reply:
But then I also have to move lines 127-143 including definition of a static array cmd_data[]. That would not work because of multiple instances of the array variable in different translation units. Function goal is to give access to the array from other source files than hci_core.c.

Reply:
Fair point :)